doc/modules/grid_search.rst

```diff
@@ -161,6 +165,21 @@ scoring function can be specified via the ``scoring`` parameter to
 specialized cross-validation tools described below.
 See :ref:`scoring_parameter` for more details.
 
+.. _multimetric_grid_search:
+
+Specifying multiple metrics for evaluation
+------------------------------------------
+
+From version 0.19, ``GridSearchCV`` and ``RandomizedSearchCV`` allows specifying
+multiple metrics for the ``scoring`` parameter.
+
+Multimetric scoring can either be specified as a list of strings of predefined
```

Comment:
Maybe add a very very short example of the standard use-case using built-in metrics?